MekHQ/src/mekhq/campaign/finances/Money.java

```diff
@@ -114,6 +118,10 @@ public Money plus(List<Money> amounts) {
     }
 
     public Money minus(Money amount) {
+        if (amount == null) {
+            return minus(0L);
```

Comment:
This ensures we get a clone of the underlying `BigMoney` instance and don't share it between two `Money` instances.